src/surfaces/IdleSurface.module.css

```
@@ -105,6 +105,40 @@
+  opacity: 0;
+}
+
+.loadingDots span:nth-child(2) {
+  animation-delay: 0.18s;
+}
+
+.loadingDots span:nth-child(3) {
+  animation-delay: 0.36s;
+}
+
+@keyframes loadingDotPulse {
```

Stylelint (17.10.0)

[error] Expected keyframe name "loadingDotPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)